package.json

```diff
@@ -239,7 +239,7 @@
 					},
 					"configmap": {
 						"type": "string",
-						"description": "Configmap key which needs to be already deployed."
+						"description": "Configmap key. The configmap is added as runtime configuration."
```

Comment:
I find this description a little confusing. The field is the "key" but the configmap is added?

Reply:
yes. the configmap corresponding to the key is added to the integration as runtime configuration.

Reply:
Might be good to spell that out then. It's not clear from the description the relationship between the key and the configmap